#
#  ### Params of the orthopus interface ###
#

#
#  Minimize cartesian velocity coefs : dx
#        
alpha_1:          10.
alpha_2:          10.
alpha_3:          10.
alpha_4:          10.
alpha_5:          10.
alpha_6:          10.
alpha_7:          10.

#
#  Minimize joints velocities coefs : dq
#        
beta_1:           0.1
beta_2:           0.1
beta_3:           0.1
beta_4:           0.1
beta_5:           0.1
beta_6:           0.1

#
#  Minimize cartesian position coefs : x
#        
gamma_1:          0.0
gamma_2:          0.0
gamma_3:          0.0
gamma_4:          5000.0
gamma_5:          5000.0
gamma_6:          5000.0
gamma_7:          5000.0


#
#  Saved positions 
#        
home_position:    [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
rest_position:    [0.0, 0.64, -1.397, 0.0, 0.0, 0.0]
drink_position:   [-1.9024448708439636, -0.3165213650985205, -0.013463968515384827, 1.543569190463785, 1.2456065805633132, -1.1438887917570835]
reference_frame             : "ground_link"
move_group_commander_name   : "arm"
allow_replanning            : true
goal_joint_tolerance        : 0.001
goal_position_tolerance     : 0.001
goal_orientation_tolerance  : 0.01
        
planning_plugin             : "ompl_interface/OMPLPlanner"
